wiki/Tournaments/CFC/3/en.md

```diff
@@ -0,0 +1,201 @@
+---
+tags:
+- Catch French Cup 3
+- CFC3
+---
+# Catch French Cup 3
+
+![CFC 3 Logo](img/logo.png)
+
+The **Catch French Cup 3 (***CFC 3***)** is a one-on-one French osu!catch tournament hosted by ![][flag_FR] [Kasumii-sama](/users/6177263). It is the 3rd installment of the Catch French Cup.
+
+## Tournament Schedule
+
+| Event | Timestamp |
+| --: | :-- |
+| Registration Phase | 2018-08-15/2018-09-02 |
+| Live Drawings | 2018-09-01 (19:00 UTC) |
+| Group Stage | 2018-09-08/2018-09-16 |
+| Round of 16 | 2018-09-22/2018-09-23 |
+| Quarterfinals | 2018-09-29/2018-09-30 |
+| Semifinals | 2018-10-08/2018-10-09 |
+| Finals | 2018-10-15/2018-10-16 |
+| Grand Finals | 2018-10-22 |
+
+## Prizes
+
+| Placing | Prize(s) |
+| :-: | :-- |
+| ![Gold Crown](/wiki/shared/GCrown.png "1st place") | 10 months of osu!supporter tag, unique profile badge |
+| ![Silver Crown](/wiki/shared/SCrown.png "2nd place") | 6 months of osu!supporter tag |
+| ![Bronze Crown](/wiki/shared/BCrown.png "3rd place") | 4 months of osu!supporter tag |
+| *4th place* | 2 months of osu!supporter tag |
+
+## Organisation
+
+The Catch French Cup 3 is run by various community members.
```

Comment:
The Catch French Cup 3 is run by various community members by distributing the multitude of tasks into various fields of responsibility.